notebooks/example/anomaly_code.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<p>How do we know this method is good? Are there any baselines to compare it with? For example, does HTM significantly outperform just flagging all changes in value greater than 3x standard deviation over a rolling window as an anomaly? Is there anything else we can compare it to to justify the claim "the model does a good job"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/HTM-applications/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>